src/sentry/static/sentry/app/views/settings/components/dataPrivacyRules/dataPrivacyRulesForm/dataPrivacyRulesFormEventIdStatusIcon.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import ControlState from 'app/views/settings/components/forms/field/controlState';
+import {t} from 'app/locale';
+import Tooltip from 'app/components/tooltip';
+import {IconClose, IconCheckmark} from 'app/icons';
+
+import {EventIdStatus} from './types';
+
+type Props = {
+  onClickIconClose: () => void;
+  status?: EventIdStatus;
+};
+
+const DataPrivacyRulesFormEventIdStatusIcon = ({status, onClickIconClose}: Props) => {
+  switch (status) {
+    case EventIdStatus.ERROR:
+    case EventIdStatus.INVALID:
+    case EventIdStatus.NOT_FOUND:
+      return (
+        <CloseIcon onClick={onClickIconClose}>
```

Comment:
I would rename this to something like IconCloseWrapper, because right now it's kind of confusing to have IconClose and CloseIcon